teams.py

```diff
@@ -40,3 +40,15 @@ def __getitem__(self, item):
 
     def __len__(self):
         return len(self.team)
+
+    def find_stronges_warrior(self):
+        pass
+
+    def find_weakest_opponent(self):
+        lowest_hp = 0
```

Comment:
Brzydko. Potem w pętli porównujesz to zawsze z zerem...
Lepiej dać tu ogromną liczbę.
Albo możesz zaimportować bibliotekę math. Ona ma coś takiego jak inf - nieskończoność.

math.inf > pierdylion zwróci True

Reply:
no myslalem o tym, ale uznalem, ze sie przyczepisz, ze wpisalem konkretna liczbe, a co bedzie jezeli walka bedzie trwala dlugo...

ale ten inf moze byc ok

Reply:
ok zrobilem chyba